src/utils/sprint-utils.ts

```diff
@@ -110,31 +110,23 @@ export const getSeveraUserId = (user: User): string => {
 };
 
 /**
- * Formating date from Severa's date format
- *
- * @param date date with type string
+ * Mapping phases to rows for datagrid
+ * 
+ * @param phase phase with type Phase
+ * @param workHours list of workHours with type WorkHours[]
  * 
- * @returns date in format of "dd MMM yyyy"
+ * @returns PhaseRow
  */
-export const formatDateSevera = (date: string) => {
-  return new Date(date).toLocaleDateString("fi-FI", {
-    day: "2-digit",
-    month: "numeric",
-    year: "numeric",
-  });
-};
-
 export const mapPhasesToRows = (phase: Phase, workHours: WorkHours[]) => {
   const actualWorkHours = getWorkHour(
     phase.severaPhaseId || "", workHours
   );
```

Comment:
```suggestion
  );
  
```

Reply:
As a general rule I think a line break before a return improves the readability of the code